README.md

```diff
@@ -26,8 +26,14 @@ $ pip install --pre django-mongodb-backend==5.1.*
 
 From your shell, run the following command to create a new Django project
 called `example` using our custom template. Make sure the zipfile referenced
-at the end of the template link corresponds to your
-version of Django. The snippet below specifies `5.1.x.zip` at the end of
+at the end of the template link corresponds to your version of Django. 
+
+You can check what version of Django you're using with:
+```bash
+$ pip show django
```

Comment:
The output of `$ django-admin --version` is a lot simpler. (However, if they follow the instructions above, I'm not sure how they would get to this point without knowing what version of Django they're using.)

Reply:
Also `pip install django-mongodb-backend` installs the corresponding Django

Reply:
Then let's have `main` be non-specific?

`pip install django-mongodb-backend`
and then also check the version. 

I've checked with some folks adhoc, they have still just done `pip install django-mongodb-backend`

Reply:
I'd rather not omit the version install advice/qualifiers, since there will be plenty of people who want to add this to an existing project on a particular version of Django. What you wrote is fine for now.